packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -2032,8 +2046,10 @@ class LookedUpVpc extends VpcBase {
   public readonly isolatedSubnets: ISubnet[];
   private readonly cidr?: string | undefined;
 
-  constructor(scope: Construct, id: string, props: cxapi.VpcContextResponse, isIncomplete: boolean) {
-    super(scope, id);
+  constructor(scope: Construct, id: string, props: cxapi.VpcContextResponse & { region?: string }, isIncomplete: boolean) {
+    super(scope, id, {
+      region: props.region ?? undefined,
```

Comment:
Same comment as above. 
```suggestion
      region: props.region,
```